docs/management/admin/blocklist.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[blocklist]]
+[chapter]
+= Blocklist
+
+coming[8.2.0]
+
+The blocklist allows you to prevent specified applications from running if you're certain they should never run in a host environment, extending the list of processes that {endpoint-sec} considers malicious. This is especially useful to ensure that known malicious processes aren't accidentally executed by an end user. 
```

Comment:
Not sure if the last sentence (the one I made suggestions on) is necessary, I think folks are pretty clear on the value of blocklisting and it extends beyond adding guardrails for users/insiders.

Reply:
I get what you mean, but I also like having a little extra explanation for the feature beyond the boilerplate description. @caitlinbetz, any thoughts? Or other use case info you think would be helpful here?

This combines @benironside's edits and other tweaks for consistency with similar wording:
```suggestion
The blocklist allows you to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. This is especially useful for ensuring that known malicious processes aren't accidentally executed by end users.
```